tests/test_models/test_engine/test_file_storage.py

```diff
@@ -113,3 +113,57 @@ def test_save(self):
         with open("file.json", "r") as f:
             js = f.read()
         self.assertEqual(json.loads(string), json.loads(js))
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
```

Comment:
Similarly to the other file, This should be models.storage_t != 'db' not models.storage_t == 'db', as we are Not testing DB in the test_db_storage.py